packages/TelegramClient-Core.package/TCCMessage.class/class/newFromMessageEvent.in..st

```diff
@@ -0,0 +1,11 @@
+instance creation
+newFromMessageEvent: aMessageEvent in: aChat
+
+	^ self new
+		message: (((aMessageEvent at: 'content') at: '@type') = 'messageText' 
+			ifTrue: [((aMessageEvent at: 'content') at: 'text') at: 'text']
+			ifFalse: ['message type not supported yet']);
+		chat: aChat;
+		senderID: ((aMessageEvent at: 'sender') at: 'user_id');
+		id: (aMessageEvent at: 'id');
+		isOutgoing: (aMessageEvent at: 'is_outgoing')
```

Comment:
Why do we even have the `senderID` attribute? The only use we found was in a test, where it could easily be replaced with `sender id`

Reply:
we do not have a user cache yet, so when receiving the message history of a chat, we cannot simply get a user without making another request and waiting for it. Due to this, the senderID inst var may be left empty